src/dfx/src/lib/canister_info/motoko.rs

```diff
@@ -75,7 +75,7 @@ impl CanisterInfoFactory for MotokoCanisterInfo {
         let output_stable_path = output_wasm_path.with_extension("most");
         let output_did_js_path = output_wasm_path.with_extension("did.js");
         let output_canister_js_path = output_wasm_path.with_extension("js");
-        let output_assets_root = output_root.join("assets");
+        let output_frontend_root = output_root.join("assets");
```

Comment:
Unfortunately search/replace `_assets` -> `_frontend` is going to pick up some false positives:

```suggestion
        let output_assets_root = output_root.join("assets");
```